scripts/findPatch.sh

```diff
@@ -1,20 +1,56 @@
 #!/bin/bash
 
 # this script should be run with cwd=project root.
-
-if [ "$1" = "" ] || [ "$2" = "" ] || [ "$3" = "" ] ｜｜ [ "$4" = "" ]; then
+if [ "$#" != 4 ]; then
     echo "Usage: ./findPatch.sh <polluter> <victim> <cleanerJsonFilePath> <mvnProjectLocalPath>"
     exit 1;
 fi
 
 
+
 mvnProjectLocalPath="$4"
 minimizedPath="$mvnProjectLocalPath/.dtfixingtools/minimized/"
 detectionResultsPath="$mvnProjectLocalPath/.dtfixingtools/detection-results/"
+orginalOrderFilePath="$mvnProjectLocalPath/.dtfixingtools/original-order.json"
+polluter="$1"
+victim="$2"
+cleanerJsonFilePath="$3"
+resultFolderPath="$mvnProjectLocalPath/.dtfixingtools/test-runs/results"
+
+echo "Confirm that a suspected cleaner is a cleaner to ensure victim passes"
+mkdir -p $mvnProjectLocalPath/.dtfixingtools/
+export cleanerName=$(jq -r '.cleaners[].testMethod' $cleanerJsonFilePath)
```

Comment:
Alternatively, maybe all of this confirmation logic should be done in the `Patch.java` file? I am ok with just adding the dependency since `jq` is available pretty much everywhere.